services/github/getLocationByUsername.js

```diff
@@ -9,7 +10,9 @@ export default async function getLocationByUsername(username) {
     lat: 0,
     lon: 0,
   };
-  const octokit = new Octokit({});
+  const octokit = new Octokit({
+    auth: serverEnv.GITHUB_API_TOKEN
+  });
   let github;
```

Comment:
To verify the functionality of your solution, please add the following line of code after making the API call. Afterward, reload the user by utilizing the API mentioned in the comment.
`logger.error(`GitHub profile information fetched for the username: ${username}`);`

Reply:
don't know how to do that exactly

Reply:
We have this line here, is that enough @raviraj-github or did you mean something else?

```js
   github = await octokit.request(`GET /users/${username}`);
    logger.info(
      `github info fetched for username: ${username} and received location ${github.data.location}`
    );
```

https://github.com/EddieHubCommunity/LinkFree/pull/7531/files#diff-f57ce9d4d71fb2f4ce5b0fc4a17d800c0b65ee8ec288c0d113a1c0c0f8604548R19-R21

Reply:
I wanted that to be just for testing locally with the Error log level. So that we will know this change is working fine. No need to commit. Like this -

![image](https://github.com/EddieHubCommunity/LinkFree/assets/11041028/e9f6fddb-a0b3-4a0f-b353-1e15777bc008)


@eddiejaoude The one you highlighted is ok for production purposes.

Reply:
> don't know how to do that exactly

Add the error log I suggested. And run the below command and check if you're getting a rate limit error.
http://localhost:3000/api/system/reload?secret=development

Reply:
See this - If you still get highlighted and error then auth token solution is either not working or the request limit has been exhausted for the hr.
![image](https://github.com/EddieHubCommunity/LinkFree/assets/11041028/e08142ce-4c3d-4e69-86fe-f9619e10c9f4)

Reply:
@raviraj-github how are you loading all the location data?
The `/api/system/reload?secret=development` endpoint does not do this

Reply:
I am getting this result on Postman  and chatgpt 
![Screenshot 2023-06-14 18 35 38](https://github.com/EddieHubCommunity/LinkFree/assets/97390674/c246c8e5-b262-4683-ae0d-e0862013659e)
![Screenshot 2023-06-14 18 36 54](https://github.com/EddieHubCommunity/LinkFree/assets/97390674/d5bbd1ea-9072-4b50-884e-4df79de19c23)

Reply:
Marking this as unresolved

Reply:
> @raviraj-github how are you loading all the location data? The `/api/system/reload?secret=development` endpoint does not do this

I have written my own js file to test it. And called that from reload api.

Reply:
thank you Ravi for checking 👍